src/main/java/org/folio/orders/utils/HelperUtils.java

```diff
@@ -235,15 +236,10 @@ public static Map<Piece.Format, Integer> calculatePiecesWithItemIdQuantity(Compo
    * @return quantity of items expected in the inventory for PO Line
    */
   public static int calculatePiecesQuantity(Piece.Format format, List<Location> locations) {
-    switch (format) {
-      case ELECTRONIC:
-        return getElectronicLocationsQuantity(locations);
-      case PHYSICAL:
-      case OTHER:
-        return getPhysicalLocationsQuantity(locations);
-      default:
-        return 0;
-    }
+    return switch (format) {
+      case ELECTRONIC -> getElectronicLocationsQuantity(locations);
+      case PHYSICAL, OTHER -> getPhysicalLocationsQuantity(locations);
```

Comment:
What default value in this case?

Reply:
This is enum so that there will be no default cases. All three cases covered
![image](https://github.com/folio-org/mod-orders/assets/113523904/6e5f09aa-0659-4d90-a0e7-5367b43d33ce)